base/reflection.jl

```diff
@@ -1218,10 +1218,12 @@ See also [`applicable`](@ref).
 julia> hasmethod(length, Tuple{Array})
 true
 
-julia> hasmethod(sum, Tuple{Function, Array}, (:dims,))
+julia> f(; orange) = orange;
+
+julia> hasmethod(f, Tuple{}, (:orange,))
```

Comment:
Very minor point, but what value would you expect someone to supply for `orange`? How about changing this to a count of the number of oranges,
```suggestion
julia> f(; oranges=0) = oranges;

julia> hasmethod(f, Tuple{}, (:oranges,))
```
and then the `apples` and `bananas` below might be more consistent too.